dts/bindings/dma/arm,dma-pl330.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+description: PL330 DMA Controller
+
+compatible: "arm,dma-pl330"
+
+include: dma-controller.yaml
+
+properties:
+    reg:
+      required: true
+    microcode:
```

Comment:
Can you provide more details on what the microcode property is.  see dts/bindings/serial/atmel,sam0-uart.yaml  for example of a property level description.

Reply:
sure, ACK